tests/features/collection/collection_content.feature

```diff
@@ -5,9 +5,10 @@ Feature: Collection content
   So that I can find the information I'm looking for.
 
   Background:
-    Given the following collection:
-      | title | Turin Egyptian Collection |
-      | state | validated                 |
+    Given the following collections:
+      | title                     | state     |
+      | Turin Egyptian Collection | validated |
+      | Turin Turkish Collection  | validated |
```

Comment:
Why this change? Leftover cruft from a previous iteration? The Turkish collection is not used in the remainder of the test so it is pointless to create it.